### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by smilerz**

### CODE REVIEW

File: MediaPlayer.py [hunk 1]
```diff
@@ -158,9 +164,30 @@ def read_track_metadata(self, track):
...
+			* *title*  (``str``)   -- Search by Track title
+
+		:return: a list of matching tracks
+		:rtype: list<sync_items.AudioTag>
+		"""
+		title = kwargs.get('title')
+		rating = kwargs.get('rating')
+		query = kwargs.get('query')
+		if title: 
+			query = "SongTitle like \"%"+ title +"%\""
```
Comment (patzm) [inline MediaPlayer.py]: I think something you might like are f-strings, e.g. 
```suggestion
			query = f'SongTitle like "%{title}%"'
```
also if you want to use `"`, you can use `'` to wrap the string

**Comment by smilerz** [comment]
- changed argparse to configargparse
- added config.ini file to minimize required command line arguments
- updated README and help files to better reflect recent changes
- added ability to connect plex with token instead of password

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ PlexAPI>=4.2.0
 fuzzywuzzy
 pypiwin32
 numpy
+configargparse
```
Comment (patzm) [inline requirements.txt]: you should add `ConfigArgParse` to the requirements file.
```suggestion
ConfigArgParse
```
https://pypi.org/project/ConfigArgParse/ this is the package you are using right?

**Review by patzm — CHANGES_REQUESTED** [review]
Cool idea!

File: requirements.txt (hunk 2 above)
Comment (smilerz) [reply requirements.txt]: yes, that's the one

**Review by patzm — APPROVED** [review]
ok cool :+1: LGTM

PS: you can re-request a review by clicking this double arrow here in the upper right in the PR overview
image

**Pull request merged**